views/orams.ejs

```diff
@@ -1,19 +1,31 @@
 <!DOCTYPE html>
-<html lang="en">
+<!--[if lt IE 8]>
+  <html class="js lt-ie8 lt-ie9" lang="en">
+<![endif]-->
+<!--[if IE 8]>
+  <html class="js lt-ie9 ie8" lang="en">
+<![endif]-->
+<!--[if IE 9 ]>
+  <html class="js ie9" lang="en">
+<![endif]-->
+<!--[if !(IE)]><!-->
+  <html class="js" lang="en">
+<!--<![endif]-->
 	<head>
     <title>ORAMS Marketplace</title>
 		<meta charSet="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1"/>
 		<link rel="shortcut icon" href="/static/media/favicon.png" type="image/x-icon" />
-
+    <link rel="stylesheet" type="text/css" href="/bundle/custom.css"/>
```

Comment:
Changing order would make it harder to override uikit?

Reply:
Good catch. Let me fix